test/helpers/TestHelper.sol

```diff
@@ -0,0 +1,282 @@
+    {
+        newPair = createLBPairFromStartIdAndBinStep(_tokenX, _tokenY, _startId, DEFAULT_BIN_STEP);
+    }
+
+    function createLBPairFromStartIdAndBinStep(IERC20 _tokenX, IERC20 _tokenY, uint24 _startId, uint16 _binStep)
+        internal
+        returns (LBPair newPair)
+    {
+        newPair = LBPair(address(factory.createLBPair(_tokenX, _tokenY, _startId, _binStep)));
+    }
+
+    function convertRelativeIdsToAbsolute(int256[] memory _relativeIds, uint24 _startId)
+        internal
+        pure
+        returns (uint256[] memory absoluteIds)
+    {
+        absoluteIds = new uint256[](_relativeIds.length);
+        for (uint256 i = 0; i < _relativeIds.length; i++) {
+            int256 id = int256(uint256(_startId)) + _relativeIds[i];
+            require(id >= 0, "Id conversion: id must be positive");
+            absoluteIds[i] = uint256(id);
+        }
+    }
+
+    function convertAbsoluteIdsToRelative(uint256[] memory _absoluteIds, uint24 _startId)
+        internal
+        pure
+        returns (int256[] memory relativeIds)
+    {
+        relativeIds = new int256[](_absoluteIds.length);
+        for (uint256 i = 0; i < _absoluteIds.length; i++) {
+            relativeIds[i] = int256(_absoluteIds[i]) - int256(uint256(_startId));
+        }
+    }
+
+    function addLiquidityAndReturnAbsoluteIds(
+        ERC20Mock _tokenX,
+        ERC20Mock _tokenY,
+        uint256 _amountYIn,
+        uint24 _startId,
+        uint24 _numberBins,
+        uint24 _gap
+    )
+        internal
+        returns (
+            uint256[] memory _ids,
+            uint256[] memory _distributionX,
+            uint256[] memory _distributionY,
+            uint256 amountXIn
+        )
+    {
+        (_ids, _distributionX, _distributionY, amountXIn) =
+            addLiquidity(_tokenX, _tokenY, _amountYIn, _startId, _numberBins, _gap);
+    }
+
+    function addLiquidityAndReturnRelativeIds(
+        ERC20Mock _tokenX,
+        ERC20Mock _tokenY,
+        uint256 _amountYIn,
+        uint24 _startId,
+        uint24 _numberBins,
+        uint24 _gap
+    )
+        internal
+        returns (
+            int256[] memory _ids,
+            uint256[] memory _distributionX,
+            uint256[] memory _distributionY,
+            uint256 amountXIn
+        )
+    {
+        uint256[] memory absoluteIds;
+        (absoluteIds, _distributionX, _distributionY, amountXIn) =
+            addLiquidity(_tokenX, _tokenY, _amountYIn, _startId, _numberBins, _gap);
+        _ids = convertAbsoluteIdsToRelative(absoluteIds, _startId);
+    }
+
+    function addLiquidity(
+        ERC20Mock _tokenX,
+        ERC20Mock _tokenY,
+        uint256 _amountYIn,
+        uint24 _startId,
+        uint24 _numberBins,
+        uint24 _gap
+    )
+        internal
+        returns (
+            uint256[] memory _ids,
+            uint256[] memory _distributionX,
+            uint256[] memory _distributionY,
+            uint256 amountXIn
```

Comment:
Inconsistent notation. Furthermore, I think we should stick to solidity's syntax (https://docs.soliditylang.org/en/latest/style-guide.html) and OZ guidelines (https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/GUIDELINES.md#solidity-conventions). Essentially removing the `_` on parameters.